src/selectors-api/selectors-api.json

```diff
@@ -0,0 +1,67 @@
+{
+    "BADGE": "badge",
+    "BREADCRUMBS": "breadcrumbs",
+    "BUTTON": {
+        "BUTTON": "btn",
+        "FAKE-BUTTON": "fake-btn"
+    },
+    "CAROUSEL": "carousel",
+    "CHECKBOX": "checkbox",
+    "COMBOBOX": "combobox",
+    "CTA-BUTTON": "cta-btn",
+    "DETAILS": "details",
+    "DRAWER-DIALOG": "drawer-dialog",
+    "EXPAND-BUTTON": "expand-btn",
+    "FIELD": "field",
+    "FILTER-BUTTON": "filter-button",
+    "FILTER-GROUP": "filter-group",
+    "FILTER-LINK": "filter-link",
+    "FILTER-MENU": {
+        "FILTER-MENU": "filter-menu", 
+        "FILTER-MENU-FORM": "filter-menu-form"
+    },
+    "FILTER-MENU-BUTTON": "filter-menu-button",
+    "FLOATING-LABEL": "floating-label",
+    "FULLSCREEN-DIALOG": "fullscreen-dialog",
+    "ICON": "icon",
+    "ICON-BTN": {
+        "ICON-BTN": "icon-btn", 
+        "ICON-LINK": "icon-link"
+    },
+    "INLINE-NOTICE": "inline-notice",
+    "LEGACY-TEXTBOX": "",
+    "LIGHTBOX-DIALOG": "lightbox-dialog",
+    "LINK": { 
+        "NAV-LINK": "nav-link", 
+        "ACTION-LINK": "action-link", 
+        "FAKE-LINK": "fake-link",
+        "LINK": ""
```

Comment:
Maybe we should remove this empty one.

Reply:
I guess I wasn't sure what they would do if they had a normal link without the classes.. maybe they would just go on the "a" selector and they wouldn't need all of this?

Reply:
Yeah, any global <a> styles are handled by the global module.